docs/user-guide/global-configurations/deployment-window.md

```diff
@@ -0,0 +1,279 @@
+
+
+### Applying Window
+
+This involves the process of applying the deployment window you created above to your application(s).
+
+1. Go to the **Apply To** tab and click the **No windows** dropdown next to the application you wish to apply deployment window(s).
+
+    ![Figure 12: Applying Windows](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/apply-window-single.gif)
+
+2. Select the deployment windows from the dropdown and click **Save Changes**.
+
+#### Bulk Apply
+
+1. If you wish to apply deployment windows to multiple applications at once, use the checkbox to select the applications. 
+
+    ![Figure 13: Selecting Apps](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/select-apps.jpg)
+
+    We recommend you to use the available filters (Application, Environment, Deployment Window) to simplify the process of application selection.
+
+    ![Figure 14a: Using Filters](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/filter-apply.jpg)
+
+    ![Figure 14b: Filtered Results](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/filter-apply-2.jpg)
+
+2. On the floating widget, click **Manage Windows**.
+
+    ![Figure 15: Manage Windows](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/manage-window.jpg)
+
+3. Click the **Add Deployment Windows** dropdown to choose the deployment window(s).
+
+    ![Figure 16: Attaching Windows to Applications](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/attach-windows.gif)
+
+4. Use the **Review Changes** option to confirm the impacted environment(s) and click **Save**.
+
+    ![Figure 17: Checking Impacted Environments](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/review-environment.gif)
+
+You can remove deployment window(s) applied to one or more apps as shown below.
+
+![Figure 18a: Removing Windows from Single App](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/remove-window-single.gif)
+
+![Figure 18b: Removing Windows from Multiple Apps](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/deployment-window/remove-window.gif)
+
+
+---
+
+## Checking Deployment Window
+
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have view-only permission or above (along with access to the environment and application) to check deployment windows.
+{% endhint %}
+
+### Overview Page
+
+The **Deployment window** section shows the deployment windows configured for each [environment](../../reference/glossary.md#environment).
```

Comment:
The Deployment window section shows the Blackout & Maintenance windows configured for each environment of the application.